tpm2/tpm2_test.go

```diff
@@ -589,6 +590,59 @@ func TestPCREvent(t *testing.T) {
 	}
 }
 
+func TestPCRExtend(t *testing.T) {
+	rw := openTPM(t)
+	defer rw.Close()
+
+    const pcr = int(16)
```

Comment:
Looks like formatting is off. Did you run `go fmt` or `goimports` on this file?

Reply:
I blame huge jetlag